### DISCUSSION TITLE
ui-test-app build error itwins 3.2 release

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#3689] [Q&A]
build:frontend and console then  this error:

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "stream": require.resolve("stream-browserify") }'
        - install 'stream-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "stream": false }

**Comment by pmconne** [comment]
Try using the `build:ci` script. @aruniverse planning to fix this?

**Reply by aruniverse** [reply]
Yes planning to. The best solution at the moment instead of running build:ci across the entire monorepo is to build:ci in the core/common dir to build an esm variant of itwin/core-common. Also to note, the warning you see is non fatal, and you can close it
